### DISCUSSION TITLE
kubecub community is recruiting new members! Join us to contribute to open source projects and improve your engineering skills! 🎉

### DISCUSSION THREAD

**Discussion by cubxxw** [kubecub/community#3] [Show and tell]
## Join the Kubecub community 🎉

👋 We welcome more developers and students to join the Kubecub community, contribute to open source projects, learn new technologies, and improve engineering capabilities. If you want to join us, you can do so in the following ways:

### Join the Kubecub project 🤝

💡 You can contribute to any project or feature module of Kubecub. Collaborative development can help improve your technical skills and engineering capabilities, and add rich experience to your resume.

### Create a new repository 🚀

📚 If you have your own project that you want to make public, you can use our scaffolding and templates to create a new repository and make it public. This can help you get a better development experience, and find more developers and friends with common goals and aspirations.

### Join Slack 💻

👨‍💻 You can join our [Slack](https://join.slack.com/t/kubecub/shared_invite/zt-1se0k2bae-lkYzz0_T~BYh3rjkvlcUqQ) to ask questions and get help from our community, as well as discuss and share your ideas and suggestions.

### How to join? 🤖

👩‍💻 If you want to join us, please comment below and use the following template:

```
English name:
GitHub username:
Email address:
City:
```

🎉 We will contact you as soon as possible. Welcome to the Kubecub community!🎊

🙌🌟👏🔥

**Comment by YaoSiQian** [comment]
English name: YaoSiQian
GitHub username: YaoSiQian
Email address: [admin@yaosiqian.cn](mailto:admin@yaosiqian.cn)
City: Shanghai

**Reply by cubxxw** [reply]
good, join slack: https://join.slack.com/t/kubecub/shared_invite/zt-1se0k2bae-lkYzz0_T~BYh3rjkvlcUqQ

**Comment by suisbuds** [comment]
I want to join kubecub
English name: suisbuds
GitHub username: suisbuds
Email address: suisbuds@gmail.com
City: Wuhan
